smoke_tests/README.md

```diff
@@ -0,0 +1,37 @@
+# smoke_tests
+
+This directory contains [artillery.io](https://artillery.io/) configuration to run post service deployment smoke tests.
+
+These tests should to run after a deployment to ensure that things are behaving as expected in the deployed environment.
+
+In addition this directory contains stress testing configuration to enable load testing when required.
+
+## Running smoke tests
+
+```
+# Install dependencies
+yarn install
+
+# AWS credentials are required to retrieve secrets 
+# in the Experience AWS account for load testing 
+# authenticated APIs
+export AWS_REGION=eu-west-1 
+export AWS_PROFILE=experience 
+
+# Run smoke tests against stage
+yarn smokeTestCatalogueApiStage
+
+# Run smoke tests against prod
+yarn smokeTestCatalogueApiProd
+```
+
+## Running stress tests
+
+**Important:** Ensure that you are not going to put undue load on the production services (or dependent Elastic clusters)! Lack of due care could break the catalogue API!
```

Comment:
How do I know I'm not going to put undue load on the prod services?

Possibly worth explaining how to interrupt a running test?

Reply:
As above, will address shortly.